src/test/java/carRacing/RacingGameTest.java

```diff
@@ -0,0 +1,42 @@
+package carRacing;
+
+import carRacing.model.RacingCar;
+import carRacing.model.WinnersName;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class RacingGameTest {
+    @Test
+    @DisplayName("자동차 경주 승자 찾기(1명)")
+    void racing_game_single_winner() {
+        RacingCar racingCar1 = new RacingCar("minn", 1);
+        RacingCar racingCar2 = new RacingCar("ssoo", 7);
+        RacingCar racingCar3 = new RacingCar("lalze", 4);
+
+        List<RacingCar> racingCarList = Arrays.asList(racingCar1,racingCar2,racingCar3);
```

Comment:
```suggestion
        List<RacingCar> racingCarList = List.of(racingCar1,racingCar2,racingCar3);
```

`Arrays.asList` 대신 `List.of` 사용을 권장드려요
자세한 내용은 [블로그](https://inpa.tistory.com/entry/JAVA-%E2%98%95-ArraysasList-%EC%99%80-Listof-%EC%B0%A8%EC%9D%B4-%ED%95%9C%EB%B0%A9-%EC%A0%95%EB%A6%AC) 읽어보시면 도움이 될 거에요